approval_frame/forms.py

```diff
@@ -17,3 +18,22 @@ class RegistrationFormCustom(RegistrationFormUniqueEmail):
         error_messages={'invalid': _("This value may contain only letters, "
             + "numbers and ./+/-/_ characters.")}
     )
+
+class PasswordResetFormCustom(PasswordResetForm):
```

Comment:
I'm not sure why we need this class. Isn't everything covered in the PasswordResetForm class?